### ISSUE TITLE
Failing lighthouse accessibility tests in foundation-layout variant due to Foundation's use of ARIA roles

### ISSUE THREAD

**Issue by eoinkelly** [ackama/rails-template#74]
We are failing Lighthouse accessibility tests in the foundation-layout variant. The issue seems to be that the navigation elements are getting a `menuitem` ARIA role and that role is required to be within an element that has one of `group, `menu`, `menubar`.

See the following for more details on the problem

* https://web.dev/aria-required-parent/?utm_source=lighthouse&utm_medium=devtools
* https://www.w3.org/TR/wai-aria-1.1/#menuitem

The challenge for us is that these roles are applied dynamically by Foundation JS so we can't easily change them without being super hacky/brittle. I found https://github.com/foundation/foundation-sites/issues/11453 which seems related and has been closed with a "Revisit for V7" label so I wouldn't expect a fix soon.

I'm not sure what the right course of action is here tbh. On the one hand, having some views which demonstrate how to use Foundation is useful but I don't want to turn off accessibility auditing. Discussion needed.

### Test failure output

```
* ******************************************************
* Running all rspec specs
* ******************************************************
.......FF
Failures:
  1) Homepage behaves like an accessible page with no errors
     Failure/Error: expect(page).to be_accessible.according_to(:wcag2a, :wcag2aa)
     
       Found 1 accessibility violation:
     
       1) aria-required-parent: Certain ARIA roles must be contained by particular parents (critical)
           https://dequeuniversity.com/rules/axe/3.5/aria-required-parent?application=axeAPI
           The following 8 nodes violate this rule:
           
               Selector: ul[data-accordion-menu="v8t6ie-accordion-menu"] > .navigation__item[role="treeitem"]:nth-child(1) > a[role="menuitem"][href="\#"]
               HTML: <a href="#" role="menuitem">Right item 1</a>
               Fix any of the following:
               - Required ARIA parents role not present: menu, menubar
               
               Selector: ul[data-accordion-menu="v8t6ie-accordion-menu"] > .navigation__item[role="treeitem"]:nth-child(2) > a[role="menuitem"][href="\#"]
               HTML: <a href="#" role="menuitem">Right item 2</a>
               Fix any of the following:
               - Required ARIA parents role not present: menu, menubar
               
               Selector: #l3x8gk-acc-menu-link > a[role="menuitem"][href="\#"]
               HTML: <a href="#" role="menuitem">Right item 3</a>
               Fix any of the following:
               - Required ARIA parents role not present: menu, menubar
               
               Selector: #f2g764-acc-menu-link > a[href="\/"][role="menuitem"]
               HTML: <a href="/" role="menuitem">Left item 1</a>
               Fix any of the following:
               - Required ARIA parents role not present: menu, menubar
               
               Selector: .navigation__item--borderless.navigation__item[role="treeitem"]:nth-child(2) > a[href="\/"][role="menuitem"]
               HTML: <a href="/" role="menuitem">Left item 2</a>
               Fix any of the following:
               - Required ARIA parents role not present: menu, menubar
               
               Selector: #\38 4sb4j-acc-menu-link > a[role="menuitem"][href="\#"]
               HTML: <a href="#" role="menuitem">Second right item 1</a>
               Fix any of the following:
               - Required ARIA parents role not present: menu, menubar
               
               Selector: .navigation__item--borderless.navigation__item[role="treeitem"]:nth-child(2) > a[role="menuitem"][href="\#"]
               HTML: <a href="#" role="menuitem">Second right item 2</a>
               Fix any of the following:
               - Required ARIA parents role not present: menu, menubar
               
               Selector: #\34 9j03z-acc-menu-link > a[role="menuitem"][href="\#"]
               HTML: <a href="#" role="menuitem">Second right item 3</a>
               Fix any of the following:
               - Required ARIA parents role not present: menu, menubar
               
       Invocation: axe.run({"runOnly":{"type":"tag","values":["wcag2a","wcag2aa"]}}, callback);
     
     [Screenshot]: /home/travis/build/ackama/rails-template/template-test/dummy/foundation-layout/tmp/screenshots/failures_r_spec_example_groups_homepage_behaves_like_an_accessible_page_with_no_errors_583.png
     
     Shared Example Group: "an accessible page" called from ./spec/system/home_feature_spec.rb:11
     # ./spec/support/shared_examples/an_accessible_page.rb:3:in `block (2 levels) in <top (required)>'
  2) Homepage behaves like an accessible page passes a Lighthouse accessibility audit
     Failure/Error: expect(page).to pass_lighthouse_audit(:accessibility)
     
       expected http://127.0.0.1:37821/ to pass Lighthouse accessibility audit
       with a minimum score of 100
     
     [Screenshot]: /home/travis/build/ackama/rails-template/template-test/dummy/foundation-layout/tmp/screenshots/failures_r_spec_example_groups_homepage_behaves_like_an_accessible_page_passes_a_lighthouse_accessibility_audit_390.png
     
     Shared Example Group: "an accessible page" called from ./spec/system/home_feature_spec.rb:11
     # ./spec/support/shared_examples/an_accessible_page.rb:7:in `block (2 levels) in <top (required)>'
Finished in 14.82 seconds (files took 4.58 seconds to load)
9 examples, 2 failures
Failed examples:
rspec ./spec/system/home_feature_spec.rb[1:2:1] # Homepage behaves like an accessible page with no errors
rspec ./spec/system/home_feature_spec.rb[1:2:2] # Homepage behaves like an accessible page passes a Lighthouse accessibility audit
```

**Comment by G-Rath** [comment]
This sounds like something that @ackama/maximum-sass should take a look at!

**Comment by eoinkelly** [comment]
This has been fixed
